lib/hammer_cli_foreman/compute_profile.rb

```diff
@@ -0,0 +1,83 @@
+require 'hammer_cli_foreman/image'
+
+module HammerCLIForeman
+  class ComputeProfile < HammerCLIForeman::Command
+    resource :compute_profiles
+    def self.add_compute_resource(compute_resource)
+      @compute_resources ||= {}
+      @compute_resources[compute_resource.name] = compute_resource
+      @compute_resources
+    end
+
+    def self.compute_resources
+      @compute_resources
+    end
+    require 'hammer_cli_foreman/compute_resources/all'
+
+    def self.get_params(options)
+      # require 'pry'; binding.pry
+      params = {}
+      params['compute_attribute'] = {}
+      profile = HammerCLIForeman.record_to_common_format(
+        HammerCLIForeman.foreman_resource(:compute_profiles).call(:show, 'id' => options['option_compute_profile_id'] )
+      )
+      HammerCLIForeman.foreman_resource(:compute_resources).call(:show, 'id' => options['option_compute_resource_id'] )
+      params['compute_attribute'] = profile['compute_attributes'].select { |hash| hash['compute_resource_id'] == options['option_compute_resource_id']}[0] || {}
+      params
+    end
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _('Id')
+        field :name, _('Name')
+      end
+      build_options
+    end
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+      output ListCommand.output_definition do
+
+        HammerCLIForeman::References.taxonomies(self)
+        HammerCLIForeman::References.timestamps(self)
+
+        collection :compute_attributes, _('Compute attributes') do
+          field :id, _('Id')
+          field :name, _('Name')
+          field nil, _('Compute Resource'), Fields::SingleReference, :key => :compute_resource
+          field :vm_attrs, _('VM attributes')
+        end
+      end
+      build_options
+    end
+
+    class CreateCommand < HammerCLIForeman::CreateCommand
+      success_message _('Compute profile created.')
+      failure_message _('Could not create a compute profile')
+      build_options
+    end
+
+    class UpdateCommand < HammerCLIForeman::UpdateCommand
+      success_message _('Compute profile updated.')
+      failure_message _('Could not update the compute profile')
+      validate_options do
+        any(:option_name,:option_id).required
+      end
+      build_options
+    end
+
+    class DeleteCommand < HammerCLIForeman::DeleteCommand
+      success_message _('Compute profile deleted.')
+      failure_message _('Could not delete the Compute profile')
+      validate_options do
+        any(:option_name,:option_id).required
+      end
+      build_options
+    end
+
+    lazy_subcommand('attribute', _("Create update and delete Compute Resource attributes"),
```

Comment:
According to our discussion `attribute` should be renamed to `values` with the description "Create, update and delete compute attributes, interfaces and volumes"